moab/meta.yaml

```diff
@@ -0,0 +1,17 @@
+package:
+  name: moab
+  version: 4.8.0
+
+source:
+  fn: moab.tar.gz
+  url: ftp://ftp.mcs.anl.gov/pub/fathom/moab-4.8.0.tar.gz
```

Comment:
I think Andy's point is moot since we are going to vet the appropriate version of MOAB before including it in this file anyway.  To some degree it doesn't really matter the quality of the naming as long as the tarball is a static snapshot.